cfpb/cfgov-refresh
# Add email signup to full width text module

On the Buying a House project, we needed a way to make email signups more visible on the journey browse pages (currently the only option is to add them in the sidefoot). This adds the ability to create email signups as insets in the FullWidthText module. (See Hubcap issue 84 for more details)

## Additions

- Email signup in full width text module
- New H3 header & inline disclaimer options in email signup module

## Notes
- We decided to make the new header/disclaimer options available in the existing email signup module instead of creating a new module for a couple of reasons -- there might be future changes to email signup headers/disclaimers across the board, and it also might be useful in the short term to have the option for inline disclaimers in sidebar signups. 
- The email signup JS isn't currently added to the page when a signup is created in the full width text module. There's a PR open in wagtail-inventory that should fix this

## Testing

1. Check out branch
2. `python cfgov/manage.py migrate` and `gulp styles`
3. In admin, create a new browse page & add a full width text module with an email signup followed by a content block
4. Verify that the signup displays as expected on page and that disclaimer/header option toggles work as expected 
5. Verify that existing sidebar signup modules get the slug header/stacked disclaimer (the defaults for these options should mean that current signups are unchanged visually)

## Screenshots

### Changes to email signup module in admin
<img width="992" alt="screen shot 2018-05-08 at 12 16 15 pm" src="https://user-images.githubusercontent.com/778171/39780124-28177e22-52c0-11e8-8f73-250d2342095b.png">

### Signup in full width text module with H3 & inline disclaimer

<img width="922" alt="screen shot 2018-05-08 at 8 29 50 am" src="https://user-images.githubusercontent.com/778171/39768402-0e1a8720-529e-11e8-889a-3ad7387e7059.png">


## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
